nucliadb/src/nucliadb/ingest/orm/brain.py

```diff
@@ -204,16 +206,15 @@ def apply_field_metadata(
                     representation=representation,
                 ),
             )
-            p.labels.append(f"/k/{Paragraph.TypeParagraph.Name(paragraph.kind).lower()}")
-
-            for classification in paragraph.classifications:
-                label = f"/l/{classification.labelset}/{classification.label}"
-                if label not in denied_classifications:
-                    p.labels.append(label)
-
-            # Add user annotated labels to paragraphs
-            extend_unique(p.labels, paragraph_classifications.valid.get(key, []))  # type: ignore
-
+            paragraph_labels = set()
+            paragraph_labels.add(f"/k/{Paragraph.TypeParagraph.Name(paragraph.kind).lower()}")
```

Comment:
You can do this all at once with the set literal syntax `{ }` if you'd like. Same in more places.

```suggestion
            paragraph_labels = {f"/k/{Paragraph.TypeParagraph.Name(paragraph.kind).lower()}"}
```